.env.example

```
@@ -383,10 +383,20 @@
 # Ratelimiting via unkey
 UNKEY_ROOT_KEY=
 
-# Used for Cal.ai Enterprise Voice AI Agents
+# Used for Cal.ai Voice AI Agents
 # https://retellai.com
 RETELL_AI_KEY=
 
+# Used for buying phone number for cal ai voice agent
+STRIPE_PHONE_NUMBER_MONTHLY_PRICE_ID=
+
+
```

dotenv-linter (3.3.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)